bin/docker-outline-dns

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Get docker ip
+docker_ip=$(ip -family inet -oneline addr show docker0|\
```

Comment:
Apology
The two files are about two differents things.
For the dns_dock version, only the change in docker-init is requiered.
The file docker-outline-dns is to enable use this system on outline mode (without connection the network manager does not link DNSDOCK as DNS server.
Do you want two differents Pull requests ? (or just one about docker-compose v2 ?)